ivy/functional/ivy/general.py

```diff
@@ -556,31 +556,139 @@ def to_scalar(x: Union[ivy.Array, ivy.NativeArray]) -> Number:
 @inputs_to_native_arrays
 def to_list(x: Union[ivy.Array, ivy.NativeArray]) -> List:
     """Creates a (possibly nested) list from input array.
-
+    
     Parameters
     ----------
     x
         Input array.
-
+        
     Returns
     -------
     ret
         A list representation of the input array ``x``.
-
+        
     Examples
     --------
+    
+    With :code:`ivy.Array` input:
+    
     >>> x = ivy.array([-1, 0, 1])
     >>> y = ivy.to_list(x)
     >>> print(y)
-    [-1, 0, 1]
-
+    ivy.to_list([-1, 0, 1])
+    
     >>> print(isinstance(y, list))
     True
-
+    
+    >>> x = ivy.array([[1.1, 2.2, 3.3], \
+                   [-4.4, -5.5, -6.6]])
+    >>> y = ivy.to_list(x)
+    >>> print(y)
+    ivy.to_list([ 1.1, 2.2, 3.3], [-4.4, -5.5, -6.6])
+    
+    >>> print(isinstance(y, list))
+    True
+    
+    >>> x = ivy.array([[[-1, 0, 1],\
+                        [1, 0, -1]], \
+                        [[1, -1, 0], \
+                        [1, 0, -1]]])
+    >>> y = ivy.to_list(x)
+    >>> print(y)
+    ivy.to_list([[[-1 0 1], [1 0 -1]], [[1 -1 0], [1 0 -1]]])
+    
+    >>> print(isinstance(y, list))
+    True
+    
+    With :code:`ivy.NativeArray` input:
+    
+    >>> x = ivy.native_array([-1, 0, 1])
+    >>> y = ivy.to_list(x)
+    >>> print(y)
+    ivy.to_list([-1 0 1])
+    
+    >>> print(isinstance(y, list))
+    True
+    
+    >>> x = ivy.native_array([[-1, 0, 1], \
+                               [-1, 0, 1], \
+                               [1,0,-1]])
+    >>> y = ivy.to_list(x)
+    >>> print(y)
+    ivy.list([[-1 0 1], [-1 0 1], [1 0 -1]])
+    >>> print(isinstance(y, list))
+    True
+    
+    >>> x = ivy.native_array([[[-1 0 1], \
+                               [1 0 -1]], \
+                              [[1 -1 0], \
+                               [1 0 -1]]])
+    >>> y = ivy.to_list(x)
+    >>> print(y)
+    ivy.list([[[-1, 0, 1], [1, 0, -1]], [[1, -1, 0], [1, 0, -1]]])
+    
+    >>> print(isinstance(y, list))
+    True
+    
+    With a mix of :code:`ivy.Container` and :code:`ivy.Array` input:
+    
+    >>> x = ivy.Container(ivy.array([-1, 0, 1]))
```

Comment:
The ivy.Container inputs should be given as a keyword argument.

Take a look at this for example:
https://github.com/unifyai/ivy/blob/e13fe80e59240af64f8f16fadfcde1823ab0ac9d/ivy/functional/ivy/losses.py#L138